### PR TITLE
Shadcn setup | Initiated design system setups

### PR DESCRIPTION
**Pull request by itSubeDibesh**
## Magic this PR Brings 🎩 

- [ ] Implement `.nvmrc` config for consistent NPM versioning
- [ ] Configured Shadcn on existing project
- [ ] Fixed Imports and implemented folder structure for design system

### CODE REVIEW

File: .nvmrc
Comment (Binabh) [inline .nvmrc:1]: As we will be mostly using this repo in new projects. We may consider working on latest Node lts version? 🤔

**Review by Binabh — COMMENTED** [review]
Added Comment on Node version. Everything else looks good to merge. 🚀 🚀

File: .nvmrc
Comment (itSubeDibesh) [reply .nvmrc:1]: Thank you for mentioning 🙇‍♂️.

**Review by rodanr — APPROVED** [review]

**Review by Binabh — APPROVED** [review]
🚀

**Review by balraaz15 — APPROVED** [review]
LGTM 🚀

**Pull request merged**